docs/Getting Started/Connecting to an Infotainment System/index.md

```diff
@@ -113,7 +113,21 @@ Once you know the IP address, you need to set the websocket `endpoint` and app `
      "nicknames": ["<app name>"]
  }
 ```
+!@
+
+@![javaSE,javaEE]
```

Comment:
There should only be one Note section, all you should change is the link, here is an example of separating the content by platform inside the note. 

```
!!! NOTE
The SDL SDK is currently supported on @![iOS]iOS 10.0!@@![android]Android 4.1 (Jelly Bean)!@@![javaSE, javaEE]Java 7 (1.7)!@@![javascript]Node 9.11.2!@ and above.
!!!

```